ILSpy.ReadyToRun/ReadyToRunDisassembler.cs

```diff
@@ -47,10 +47,19 @@ public ReadyToRunDisassembler(ITextOutput output, ReadyToRunReader reader, Runti
 
 		public void Disassemble(PEFile currentFile, int bitness, ulong address, bool showMetadataTokens, bool showMetadataTokensInBase10)
 		{
-			// TODO: Decorate the disassembly with GCInfo
 			ReadyToRunMethod readyToRunMethod = runtimeFunction.Method;
 			WriteCommentLine(readyToRunMethod.SignatureString);
 
+			if (ReadyToRunOptions.GetIsShowGCInfo(null) && readyToRunMethod.GcInfo != null)
```

Comment:
Also wondering if it's possible that `readyToRunMethod.GcInfo` returns null, in that case should we show a comment that the GC Info was not found (if the checkbox was enabled)?